__tests__/app.test.js

```diff
@@ -128,6 +129,28 @@ describe("PATCH /api/reviews/:review_id", () => {
         });
       });
   })
+  test("should return status 200 and the review with vote incremented by 1 (default) if no key was sent", () => {
+    return request(app)
+      .patch("/api/reviews/4")
+      .send()
+      .expect(200)
+      .then((res) => {
+        const review = res.body.review;
+        expect(review).toEqual({
+          review_id: 4,
+          title: "Dolor reprehenderit",
+          category: "social deduction",
+          designer: "Gamey McGameface",
+          owner: "mallionaire",
+          review_body:
+            "Consequat velit occaecat voluptate do. Dolor pariatur fugiat sint et proident ex do consequat est. Nisi minim laboris mollit cupidatat et adipisicing laborum do. Sint sit tempor officia pariatur duis ullamco labore ipsum nisi voluptate nulla eu veniam. Et do ad id dolore id cillum non non culpa. Cillum mollit dolor dolore excepteur aliquip. Cillum aliquip quis aute enim anim ex laborum officia. Aliqua magna elit reprehenderit Lorem elit non laboris irure qui aliquip ad proident. Qui enim mollit Lorem labore eiusmod",
+          review_img_url:
+            "https://images.pexels.com/photos/278918/pexels-photo-278918.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260",
+          created_at: "2021-01-22T11:35:50.936Z",
+          votes: 8,
+        });
+      });
+  });
```

Comment:
In case of a missing patch object we should either get an error or a 200, but definitely not increment by 1. Votes should remain the same unless an increment value is specified in the patch object.